### PR TITLE
Feature/pollinterval

### PR DESCRIPTION
**Pull request by rdehuyss**

### CODE REVIEW

File: core/src/test/java/org/jobrunr/jobs/RecurringJobTest.java [hunk 1]
```diff
@@ -157,14 +159,26 @@ void nextInstantWithIntervalIsCorrect() {
...
+
+        assertThatThrownBy(() -> recurringJob1.validateSchedule(ofSeconds(5))).isInstanceOf(IllegalArgumentException.class);
+        assertThatThrownBy(() -> recurringJob2.validateSchedule(ofSeconds(10))).isInstanceOf(IllegalArgumentException.class);
+
+        assertThatCode(() -> recurringJob1.validateSchedule(ofMillis(200))).doesNotThrowAnyException();
+        assertThatCode(() -> recurringJob2.validateSchedule(ofSeconds(5))).doesNotThrowAnyException();
     }
 
     @Test
     void smallestIntervalForRecurringIntervalJobIs5Seconds() {
```
Comment (auloin) [inline core/src/test/java/org/jobrunr/jobs/RecurringJobTest.java:174]:
```suggestion
    void testValidateScheduleForRecurringIntervalJob() {
```

**Review by auloin — CHANGES_REQUESTED** [review]

File: core/src/main/java/org/jobrunr/server/BackgroundJobServer.java [hunk 2]
```diff
@@ -338,6 +345,14 @@ private BackgroundJobPerformerFactory loadBackgroundJobPerformerFactory() {
                 .orElseGet(BasicBackgroundJobPerformerFactory::new);
     }
 
+    private boolean pollIntervalIsTooSmallForStorageProvider(Duration pollInterval, StorageProvider storageProvider) {
```
Comment (auloin) [inline core/src/main/java/org/jobrunr/server/BackgroundJobServer.java:348]: Is it still used?

**Review by auloin — APPROVED** [review]
Already good to merge (the one remark is about a method I believe is no longer used). Good job!

File: core/src/main/java/org/jobrunr/server/BackgroundJobServer.java (hunk 2 above)
Comment (rdehuyss) [reply core/src/main/java/org/jobrunr/server/BackgroundJobServer.java:348]: Nope, thanks. I'll remove it.

**Pull request merged**